src/mainviews/job-detail.tsx

```diff
@@ -1,13 +1,398 @@
+              aria-controls="panel-content"
+              id="panel-header"
+            >
+              <FormLabel component="legend">
+                {trans.__('Additional options')}
+              </FormLabel>
+            </AccordionSummary>
+            <AccordionDetails id="panel-content">
+              <Stack spacing={4}>
+                <TextFieldStyled
+                  label={trans.__('Output Prefix')}
+                  defaultValue={job?.output_prefix ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('Idempotency Token')}
+                  defaultValue={job?.idempotency_token ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('Job Definition Id')}
+                  defaultValue={job?.job_definition_id ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <FormLabel component="legend">{trans.__('Tags')}</FormLabel>
+                {job?.tags &&
+                  job?.tags.map(tag => (
+                    <TextFieldStyled
+                      label={trans.__('Tag')}
+                      defaultValue={tag}
+                      InputProps={{
+                        readOnly: true
+                      }}
+                    />
+                  ))}
+                <FormLabel component="legend">
+                  {trans.__('Email Notifications')}
+                </FormLabel>
+                <TextFieldStyled
+                  label={trans.__('Job Definition Id')}
+                  defaultValue={job?.timeout_seconds?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('max_retries')}
+                  defaultValue={job?.max_retries?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('min_retry_interval_millis')}
+                  defaultValue={
+                    job?.min_retry_interval_millis?.toString() ?? ''
+                  }
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('retry_on_timeout')}
+                  defaultValue={job?.retry_on_timeout?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('start_time')}
+                  defaultValue={job?.start_time?.toString() ?? ''}
```

Comment:
This should be converted from epoch seconds to a readable format (see the existing list view for more about how we do this)

Reply:
Feel free to open another issue if this is too complex to do right now.

Reply:
Adapted your solution, otherwise would probably be too complex for now